src/site/content/en/blog/badging-api/index.md

```diff
@@ -0,0 +1,224 @@
+---
+title: Badging for App Icons
+subhead: The Badging API allows installed web apps to set an application-wide badge, shown in an operating-system-specific place associated with the application, such as the shelf or home screen.
+authors:
+  - petelepage
+description: The Badging API allows installed web apps to set an application-wide badge, shown in an operating-system-specific place associated with the application, such as the shelf or home screen. Badging makes it easy to subtly notify the user that there is some new activity that might require their attention, or it can be used to indicate a small amount of information, such as an unread count.
+date: 2018-12-11
+updated: 2019-09-26
+tags:
+  - post
+  - capabilities
+  - badging
+  - progressive-web-apps
+  - notifications
+  - origintrials
+  - fugu
+hero: hero.jpg
```

Comment:
I pushed up a version of the hero with a lower compression ratio, made the image bigger, but seems better.

Reply:
I'll defer to @robdodson to decide

Reply:
The quality of the image seemed ok to me. I think we set the dimensions at 1600 x 480, but it might be good to do these as 3200 x 960 so we can have 2x versions. We'll be switching over to imgix soon which will only serve the correct res version using srcset.

Reply:
The [handbook](https://web.dev/handbook/markup-media/#hero-images) states 1600x800, which is what this is.

Reply:
Yeah I just realized we need to update that guidance. I'll do that now. You can leave it at 1600 if you want, but if you want a crisper version for mobile then a 3200 might look nice.

Reply:
Updated the handbook.

For additional context. Most of the existing hero images are 1600, but we're switching over to an image cdn which will let us srcset the images better. So adding a higher res one now is mostly a future proofing thing if you want to take advantage of 2x on desktop displays. I realize how annoying that is because you already generated your hero images :( It's up to you if you want to redo them—most of the other posts on the site have 1600 heros and will continue to do so after we switch to the cdn because it feels like a nightmare to go back and redo all of those.